app.ts

```diff
@@ -101,6 +102,37 @@ async function main(): Promise<void> {
       // If the user specified the 'edit' command, this function will be executed
       handler: EditHandler.handle,
     } as CommandModule<{}, EditHandler.Args>)
+    .command({
+      command: "delete",
+      describe: "Delete a task",
+      builder: (yargs) =>
+        // Configuration for the options the 'add' command accepts
```

Comment:
This comment seems to be off. Maybe it wasn't updated when copied here?

Reply:
I may have added for myself to understand yargs better in the beginning of implementing this app, but I deleted similar comments since now they are not  needed!